### PR TITLE
Add prefix: creativebiolabs

### PR DESCRIPTION
**Pull request by nagutm**
This pull request addresses the following issue: https://github.com/biopragmatics/bioregistry/issues/1257

Creative Biolabs provides identifiers for various antibody related entities. Their website lists the following categories of products that have unique identifiers which I decided to break into 2 broad semantic spaces: 
- `creativebiolabs.antibody`: Recombinant antibodies, Functional antibodies, AbPlus, Formats, Research area, Engineered antibodies, AbProcessing
- `creativebiolabs.protein`: Tetramers, Proteolytic enzymes, Antibody-like Scaffold proteins, Therapeutic protein, Soluble T Cell receptor

I also considered further dividing these into 5 semantic subspaces to include `creativebiolabs.enzyme`, `creativebiolabs.processing` and `creativebiolabs.research` and even leaving it as a single all encompassing prefix. 

I was unable to track down a specific point of contact for this resource and have currently left in some placeholder information for the original contributor of this resource as well.

### CODE REVIEW

File: src/bioregistry/data/bioregistry.json [hunk 1]
```diff
@@ -22211,6 +22211,58 @@
...
+  "creativebiolabs.antibody": {
+    "contact": {
+      "email": "info@creative-biolabs.com",
+      "name": "Creative Biolabs Team"
+    },
+    "contributor": {
+      "email": "placeholder@example.com",
+      "github": "garybrandam",
+      "name": "Gary",
+      "orcid": "0000-0000-0000-0000"
```
Comment (bgyori) [inline src/bioregistry/data/bioregistry.json:22223]: @garybrandam we require ORCIDs for contributions and can't proceed with merging the additions unless you can provide an ORCID. We also generally require emails for contributors. Please let us know and we can add it!

File: src/bioregistry/data/bioregistry.json [hunk 2]
```diff
@@ -22211,6 +22211,58 @@
     "name": "Crates.io",
     "uri_format": "https://crates.io/crates/$1"
   },
+  "creativebiolabs.antibody": {
+    "contact": {
+      "email": "info@creative-biolabs.com",
+      "name": "Creative Biolabs Team"
```
Comment (bgyori) [inline src/bioregistry/data/bioregistry.json:22217]: @garybrandam the Bioregistry expects a specific point of contact person for each prefix. Could you provide contact info for an appropriate person, ideally name, ORCID, email, GitHub handle if available?

**Comment by bgyori** [comment]
@cthoyt since @garybrandam is not responding but the resource is still worth adding, we were planning to switch to @nagutm as contributor and finalize the PR.

**Review by cthoyt — CHANGES_REQUESTED** [review]
Please remove the contact - info@creative-biolabs.com - before merging, again, because the Bioregistry policy is only to include explicit single point of contacts.

**Comment by bgyori** [comment]
> I'm skeptical about adding these since the contributor isn't responsive, there isn't a way to resolve these IDs, and I guess there isn't a way to get a full list of identifiers.
> 
> I'm curious what makes you think it's worth adding. Have you seen these identifiers appear in some other data resource? Is it valuable just for the fact that it addresses a community suggestion? Or maybe something else?

I don't have a strong feeling about it, it's just that (1) someone explicitly suggested it (2) the work of curating the resource is already done and (3) it appears to be a legitimate, and notable resource, in particular as a producer of antibodies, so being able to refer to their catalogue numbers via CURIEs in a principled way seems like a nice thing to have for the community in general.

> 
> There are various places where I've written notes on my thought process for what makes something worthy but I don't think I've prepare a centralized guide before.
> 
> If you can take care of deleting the group email then this is good enough for now.

**Pull request merged**